web/src/emoji_picker.ts

```diff
@@ -229,9 +251,10 @@ export function is_emoji_present_in_text(text, emoji_dict) {
     return false;
 }
 
-function filter_emojis() {
-    const $elt = $(".emoji-popover-filter").expectOne();
-    const query = $elt.val().trim().toLowerCase();
+function filter_emojis(): void {
+    const $elt = $<HTMLInputElement>(".emoji-popover-filter").expectOne();
```

Comment:
Apologies if I missed to mention but in these the selector string is changed from `.foo` to `input.foo` to ensure runtime safety . So 
```
const $elt = $<HTMLInputElement>("input.emoji-popover-filter").expectOne();
```

Reply:
Yeah sure, I have made the changes!